resource-managers/kubernetes/integration-tests/scripts/setup-integration-test-env.sh

```diff
@@ -118,7 +118,7 @@ then
 
   # Build SparkR image
   tags=(${EXCLUDE_TAGS//,/ })
-  if [[ ! ${tags[@]} =~ "r" ]]; then
+  if [[ ! " ${tags[*]} " =~ " r " ]]; then
```

Comment:
cc @dongjoon-hyun. Could you look at this?
It seems that we run k8s integratation tests with `test.exclude.tags=minikube,r,local`, and the logic here actually means to check if one of the arguments is "r"? If this, then `if [[ ! " ${tags[*]} " =~ " r " ]]; then` can work well with it.